src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/util/TestStorageHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+
+public class TestStorageHelper {
+
+    private static TestStorageHelper INSTANCE;
+
+    public static final String CONTAINER_NAME = "bulkscan";
+    public static final String ZIP_FILE_NAME = "1_24-06-2018-00-00-00.zip";
+
+    private static DockerComposeContainer<?> dockerContainer;
+    public static CloudBlobClient cloudBlobClient;
+    private CloudBlobContainer testContainer;
+
+    private TestStorageHelper() {
+        // empty constructor
+    }
+
+    public static TestStorageHelper getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new TestStorageHelper();
+        }
+
+        return INSTANCE;
+    }
+
+    private static void createDocker() {
+        dockerContainer = new DockerComposeContainer<>(
+            new File("src/integrationTest/resources/docker-compose.yml")
+        ).withExposedService("azure-storage", 10000);
+        dockerContainer.start();
+    }
+
+    private static void initializeStorage() {
+        try {
+            cloudBlobClient = CloudStorageAccount
+                .parse("UseDevelopmentStorage=true")
+                .createCloudBlobClient();
+        } catch (InvalidKeyException | URISyntaxException exception) {
+            throw new RuntimeException("Unable to setup docker environment", exception);
```

Comment:
docker environment ? do we need try catch here ?

Reply:
try catch so to reduce exception signatures in test classes

Reply:
ok is this message valid `Unable to setup docker environment` ?

Reply:
Now that method name is different, will replace with `unable to initialise storage` ;)